pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -1576,15 +1576,8 @@ async def run(self):
                     content_q_by_type[model_type] = content_q_by_type[model_type] | unit_q
                     d_content_by_nat_key[d_content.content.natural_key()].append(d_content)
 
+            modified_rslts_by_pid = {}  # hold the to-be-saved results by-pid
```

Comment:
Why `pid` over `pk` and `rslts` over `content`?  I would prefer to keep using the standard conventions if it's not a problem

Reply:
Will chg to pk. I'm using "rslts" because the entities are called "result" in the loop we're going through.